charts/celeborn/templates/master-statefulset.yaml

```diff
@@ -20,21 +20,13 @@ kind: StatefulSet
 metadata:
   name: {{ .Release.Name }}-master
   labels:
-    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version }}
-    app.kubernetes.io/name: {{ .Chart.Name }}
-    app.kubernetes.io/instance: {{ .Release.Name }}
-    app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
-    app.kubernetes.io/managed-by: {{ .Release.Service }}
-    app.kubernetes.io/role: master
     {{- include "celeborn.labels" . | nindent 4 }}
+    app.kubernetes.io/role: master
 spec:
   selector:
     matchLabels:
-      app.kubernetes.io/name: {{ .Chart.Name }}
-      app.kubernetes.io/instance: {{ .Release.Name }}
-      app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
-      app.kubernetes.io/role: master
       {{- include "celeborn.selectorLabels" . | nindent 6 }}
```

Comment:
celeborn.selectorLables doesn't contain app.kubernetes.io/version..

Reply:
When creating a new Helm chart, the default `selectorLabels` should include only `app.kubernetes.io/name` and `app.kubernetes.io/instance`, as these are sufficient for distinguishing between different deployments of the chart.

Reply:
> When creating a new Helm chart, the default `selectorLabels` should include only `app.kubernetes.io/name` and `app.kubernetes.io/instance`, as these are sufficient for distinguishing between different deployments of the chart.

Make sense, thanks!